universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java

```diff
@@ -44,27 +47,160 @@ public Content render(
       long applicantId,
       ImmutableList<ProgramDefinition> programs,
       Optional<String> banner) {
-    String applyMessage = messages.at("button.apply");
-    ContainerTag body = body();
+    ContainerTag body =
+        body().withClasses(Styles.RELATIVE, Styles.PX_8, ApplicantStyles.BODY_BACKGROUND);
     if (banner.isPresent()) {
       // TODO: make this a styled toast.
       body.with(p(banner.get()));
     }
-    return layout.render(
-        body.with(h1(messages.at("title.programs")))
-            .with(each(programs, program -> shortProgram(applicantId, applyMessage, program))));
+    body.with(
+        nav()
+            .withClasses(
+                Styles.PT_8,
+                Styles.PB_4,
+                Styles.MB_12,
+                Styles.FLEX,
+                Styles.ALIGN_MIDDLE,
+                Styles.BORDER_B_4,
+                Styles.BORDER_WHITE)
```

Comment:
question: will we eventually put these in some kind of named style, like how you did with the admin nav?

Reply:
Yes, but I tend to wait until things normalize a bit before factoring out the styles. (I just assume that things will change 2-3 times before settling.)